index.php

```diff
@@ -1,27 +1,50 @@
+<?php
+
+use Collection\Models\BookModel;
+
+$db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+require_once 'src/display_books_list.php';
+require_once 'vendor/autoload.php';
+?>
+
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <title>Template</title>
+    <title>My Library</title>
 
     <meta name="description" content="Template HTML file">
     <meta name="author" content="iO Academy">
-
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
     <link rel="stylesheet" href="css/styles.css">
-
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Bricolage+Grotesque:opsz,wght@10..48,200;10..48,400&display=swap" rel="stylesheet">
     <link rel="icon" href="images/favicon.png" sizes="192x192">
     <link rel="shortcut icon" href="images/favicon.png">
     <link rel="apple-touch-icon" href="images/favicon.png">
-
     <script defer src="js/index.js"></script>
 </head>
 
 <body>
-
-<h1>Website Template</h1>
-
+    <h1>My Library</h1>
+    <?php
+    $bookModel = new BookModel($db);
+    $books = $bookModel->getAllBooks();
+    $booksList =  display_books_list($books);
+
+    if ($booksList) {
```

Comment:
Because your display\_books\_list function throws an exception, I don't actually think this if statement works. 

There are two options:

1.  Change the funciton so it returns false instead of throwing an exception
2.  Use a try-catch block here instead of an if statement (we haven't looked at them before) - they look like this:

```php
try {
// Any code that runs in here can safely throw exceptions without 'crashing' the program
// In your case, you would put your call to display_books_list() in the try

} catch (Exception $e) {
// If any code in the try portion threw an error, it gets caught in here, allowing you do do something
// about the error, like displaying your 'no results found' message
}
```